src/components/ViewRoutingSlip/LinkedRoutingSlipsPaymentInfo.vue

```diff
@@ -0,0 +1,31 @@
+<template>
```

Comment:
This is WIP component. Would be adding code for displaying children payment information in here